teamtemp/templates/about.html

```diff
@@ -1,27 +1,30 @@
 {% extends "base.html" %}
 
-{% block title %}About Team Temperature{% endblock %}
+{% block title %}About Team Temp{% endblock %}
 
 {% block body %}
-
+<div class="container">
```

Comment:
Wasn't sure if I wanted to force every page to use a container or whether there would be a scenario where the body was inside a container-fluid as opposed to a container. At the moment we don't have that scenario.

Happy to extract out the container declaration into the base and should the moment come for us ever to need a fluid one we can refactor.

Will submit some changes shortly.